.ci/patches/register-service-provider-and-facade.patch

```diff
@@ -14,7 +14,7 @@ index 1941d7c..039b0d4 100644
      */
  
      'aliases' => Facade::defaultAliases()->merge([
--        // ...
+-        // 'ExampleClass' => App\Example\ExampleClass::class,
 +        'Bugsnag' => Bugsnag\BugsnagLaravel\Facades\Bugsnag::class,
```

Comment:
Seems odd that the line numbers (191 vs [212](https://github.com/laravel/laravel/blob/master/config/app.php#L212)) don't match, but if it works then its no big deal?

Reply:
Ah I didn't notice that, but it looks like `patch` is resilient against the line numbers changing as long as the content matches:

```
patching file config/app.php
Hunk #1 succeeded at 185 (offset 18 lines).
Hunk #2 succeeded at 210 (offset 18 lines).
```

Reply:
ah, today I learned